### DISCUSSION TITLE
how can i Initialize the SDK manually on React-Native

### DISCUSSION THREAD

**Discussion by qilovehua** [getsentry/sentry-react-native#2690] [Q&A]
As China android market privacy policy，no sdks can call stack before user agree privacy policy.
Sentry.init({dns: '*'}) will call getInstallerPackageName which is not be permit.
Is there an option to disable to call getInstallerPackageName when sentry init? 
or how can I initialize sentry manually after user agree provacy policy.

md5=3CC31CEC883930AFF4F60869D4729F41,pkg=com.app,action=android.permission.MY_READ_INSTALLED_PACKAGES,content=查询包名,输出结果:null,callstack:android.app.ApplicationPackageManager.getInstallerPackageName:2057;io.sentry.android.core.DefaultAndroidEventProcessor.getSideLoadedInfo:865;io.sentry.android.core.DefaultAndroidEventProcessor.loadContextData:116;io.sentry.android.core.DefaultAndroidEventProcessor.lambda$new$0$io-sentry-android-core-DefaultAndroidEventProcessor:98;io.sentry.android.core.DefaultAndroidEventProcessor$$ExternalSyntheticLambda0.call:2;java.util.concurrent.FutureTask.run:266;java.util.concurrent.ThreadPoolExecutor.runWorker:1167;java.util.concurrent.ThreadPoolExecutor$Worker.run:641;java.lang.Thread.run:764;

**Comment by krystofwoldrich** [comment]
You could initialize the Android SDK manually after the user agrees to the privacy policy.

https://docs.sentry.io/platforms/react-native/manual-setup/native-init/

**Reply by qilovehua** [reply]
Thank you very much.
I try to call native module to initialize Android SDK manually and it works.
